src/common/simple_spin.cc

```diff
@@ -18,9 +18,12 @@
 #include <stdint.h>
 #include <pthread.h>
 
-static uint32_t bar = 13;
-static uint32_t *foo = &bar;
- 
+#if (__arm__ || __aarch64__)
```

Comment:
better off using 
```c
#if defined(__arm__) || defined(__aarch64)
```
and might be better just put this inline assembly in `simple_spin_lock()` instead of introducing another macro.